### ISSUE TITLE
Unclear / misleading docs on emptyDir volume creation

### ISSUE THREAD

**Issue by dancer1325** [kubernetes/website#43805]
**This is a Bug Report**
* Unclear / misleading information about the emptyDir volume first creation

---

**Problem:**
In https://kubernetes.io/docs/concepts/storage/volumes/#emptydir, the first line says

> An `emptyDir` volume is first created when a Pod is assigned to a node

My understanding based on that statement is that once a Pod is assigned to a node -> automatically  an `emptyDir` volume is created (not ❓ )

Dummy sample to check it

```
apiVersion: v1
kind: Pod
metadata:
  name: pod-emptydir-nospecified
spec:
  containers:
    - name: container-emptydir-nospecified
      image: nginx:latest
```

`kubectl describe pods/pod-emptydir-nospecified` should display an existing volume attached to the pod? No `emptyDir` volume found
![image](https://github.com/kubernetes/website/assets/39351487/6d118d39-bd8b-441a-a062-cfade1b28008)

---

**Proposed Solution:**
* Remove that wrong line?

---

**Page to Update:**
https://kubernetes.io/docs/concepts/storage/volumes/#emptydir

**Comment by rohan-kapse** [comment]
@dancer1325 sorry could u elaborate further.

**Comment by dancer1325** [comment]
> @dancer1325 sorry could u elaborate further.

Hi @rohan-kapse, 

>My understanding based on that statement is that once a Pod is assigned to a node -> automatically an emptyDir volume is created (not ❓ )

**Comment by rohan-kapse** [comment]
@dancer1325 yeah, but is getting the wrong line removed a right solution?

**Comment by tengqm** [comment]
The sentence can be revised to something like:

> For a Pod that defines an `emptyDir` volume, the volume is created when the Pod is assigned to a node.

**Comment by dipesh-rawat** [comment]
Fixed by #43815

/close
